src/components/Table/Table.scss

```diff
@@ -5,12 +5,14 @@
 // --------------------------------------------------
 // Data table styles
 
-
+table,
 .ms-Table {
   display: table;
   width: 100%;
+  border-collapse: collapse;
 }
 
+tr,
```

Comment:
Styling all of these elements makes sense, but we should maintain the opt-in spirit of Fabric by prefixing the selectors with .ms-Table, e.g. `ms-Table tr`.